app/views/katello/api/v2/repositories/show.json.rabl

```diff
@@ -5,7 +5,7 @@ extends 'katello/api/v2/common/timestamps'
 
 attributes :content_type
 attributes :docker_upstream_name
-attributes :unprotected, :full_path, :checksum_type, :container_repository_name
+attributes :unprotected, :full_path, :checksum_type, :container_repository_name, :download_policy
```

Comment:
I d argue all repository attributes should show up in the json,rabl.  Aren't our rabl's mapped 1:1 with the object ?.